vsintegration/src/FSharp.VS.FSI/sessions.fs

```diff
@@ -360,6 +359,49 @@ type FsiSession(sourceFile: string) =
     /// interrupt timeout in miliseconds
     let interruptTimeoutMS = 1000
 
+    let splitPidFile =
+        lazy
+            let mutable trueProcessPid = if usingNetCore then None else Some cmdProcessPid
+            let mutable trueFrameworkVersion = Single.MaxValue
+            let getTfmNumber (v: string) =
+                let arr = v.Split([| '.' |], 3)
+                match Single.TryParse(arr[0] + "." + arr[1]) with
```

Comment:
"." is only parsed as decimal separator in some cultures.
Better to pass in an explicit invariant or en culture.

(e.g. Czech has comma as a decimal separator, and dot as a thousand separator)

Reply:
I don't think it's a decimal separator, but a tfm version, whcuh is just a custom version format, much like the semver.

Reply:
It's the dotnet version and not a number.  The other thing is the actual version that value is the specific version that the behaviour changed, it's not really variable.  I could encode it any number of ways, this is just the most straightforward.

Reply:
t's the curse of working in the US. I will fix the culture.  The string is lazy because of:

        if **sessions.SupportsInteractivePrompt** then

The actual value of this relies on a new instance of fsi having being computed.
So ... it is false until that happens.